master/pom.xml

```diff
@@ -1074,6 +1074,11 @@
         <version>${sakai.version}</version>
         <scope>provided</scope>
       </dependency>
+		<dependency>
```

Comment:
The indentation here looks wrong, could you try to match it with the surrounding context, the issue maybe because of differences tabs and spaces.